### DISCUSSION TITLE
키글챌 22회차 반성문

### DISCUSSION THREAD

**Discussion by sososo0** [BDD-CLUB/challenge-every-week-posting#240] [6. 반성문]
변명이라도 해보자면... 제가 요새 카테캠 마무리 한다고... 바빠서요.. 제가 또 조장이라 신경을 아예 안 쓸 수 없어서 밤낮 가리지 않고 하다보니... 이렇게... 까마득하게 잊어먹고 반성문을 쓰고 있네요.. 앞으로는!!! 열심히 쓰겠습니다!! 아자아자

**Comment by shkisme** [comment]
화이팅입니다 !!!
